### PR TITLE
Copy assets only when necessary, and remove rootfs tar files from fil…

### PR DESCRIPTION
**Pull request by MatthewTighe**
…esystem local support directories.

**Describe the pull request**

Provide a description of the problem your pull request solves. Screenshots, code examples, etc are welcome in this section.

**Link to relevant issues**

### CODE REVIEW

File: app/src/main/java/tech/ula/utils/DownloadUtility.kt [hunk 1]
```diff
@@ -37,87 +38,99 @@ class DownloadUtility(val context: Context, val session: Session, val filesystem
             "$distType:startSSHServer.sh" to "https://github.com/CypherpunkArmory/UserLAnd-Assets-$distType/raw/$branch/assets/all/startSSHServer.sh",
             "$distType:startVNCServer.sh" to "https://github.com/CypherpunkArmory/UserLAnd-Assets-$distType/raw/$branch/assets/all/startVNCServer.sh",
             "$distType:startVNCServerStep2.sh" to "https://github.com/CypherpunkArmory/UserLAnd-Assets-$distType/raw/$branch/assets/all/startVNCServerStep2.sh",
-            "$distType:extractFilesystem.sh" to "https://github.com/CypherpunkArmory/UserLAnd-Assets-$distType/raw/$branch/assets/all/extractFilesystem.sh",
+            "$distType:extractFilesystem.sh" to "https://github.com/CypherpunkArmory/UserLAnd-Assets-$distType/raw/test/assets/all/extractFilesystem.sh",
```
Comment (corbinlc) [inline app/src/main/java/tech/ula/utils/DownloadUtility.kt]: test -> $branch

File: app/src/main/java/tech/ula/ServerService.kt [hunk 2]
```diff
@@ -262,13 +264,13 @@ class ServerService : Service() {
...
         launchAsync {
-            asyncAwait { assetsWereDownloaded = downloadAssets() }
+            asyncAwait { assetsWereDownloaded = downloadAssets(updateIsBeingForced = true) }
             killProgressBar()
             if (!assetsWereDownloaded) {
                 Toast.makeText(this@ServerService, R.string.no_assets_need_updating, Toast.LENGTH_LONG).show()
                 return@launchAsync
             }
-            fileUtility.copyDistributionAssetsToFilesystem(filesystem.id.toString(), filesystem.distributionType)
+            fileUtility.copyDistributionAssetsToFilesystem(lastActivatedFilesystem.id.toString(), lastActivatedFilesystem.distributionType)
```
Comment (corbinlc) [inline app/src/main/java/tech/ula/ServerService.kt]: lastActivatedFilesystem->filesystem

File: app/src/main/java/tech/ula/utils/DownloadUtility.kt [hunk 3]
```diff
@@ -37,87 +38,99 @@ class DownloadUtility(val context: Context, val session: Session, val filesystem
...
-            val now = System.currentTimeMillis()
-            val lastUpdateCheck = prefs.getLong("lastUpdateCheck", 0)
-            if (!asset.exists() || now > (lastUpdateCheck + TimeUnit.DAYS.toMillis(1))) {
-                with(prefs.edit()) {
-                    putLong("lastUpdateCheck", now)
-                    apply()
-                }
-            } else {
-                return false
+    // TODO make corbin rename repo to support
```
Comment (corbinlc) [inline app/src/main/java/tech/ula/utils/DownloadUtility.kt]: i already renamed the repo to support

**Comment by MatthewTighe** [comment]
@corbinlc Updated for review.

**Pull request merged**